react-native/react/native/remote-connect.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from '../base-react'
+import ipc from 'ipc'
+
+function remoteDispatch (action) {
+  if (action.constructor === Function) {
+    throw new TypeError("Can't remoteDispatch on a function!")
```

Comment:
there's actually no reason why remoteDispatch can't be a function, I just didn't realize that when I wrote this.